### DISCUSSION TITLE
how to change the default language?

### DISCUSSION THREAD

**Discussion by Heus-Sueh** [sodaliterocks/sodalite#76] [Q&A] [answered]
I click there (Portuguese) but nothing changes :thinking: 
![Screenshot from 2023-02-24 16 36 32](https://user-images.githubusercontent.com/98482469/221275037-12950587-390b-476f-b7be-e9bfb16497b2.png)

**Comment by electricduck** [comment, marked as answer]
The Locale plug is missing on Fedora Linux and that's where that is trying to redirect to. You can instead use the **Locale Selector** app found in the Applications menu to set it.

![image](https://user-images.githubusercontent.com/1594644/221431711-fc62d852-dee8-4944-91ad-edb15b1c950f.png)
